src/test/scala/com/cognite/sdk/scala/common/WritableBehaviors.scala

```diff
@@ -154,25 +157,36 @@ trait WritableBehaviors extends Matchers { this: FlatSpec =>
           missingIds should have size externalIdsThatDoNotExist.size.toLong
           missingIds should contain theSameElementsAs externalIdsThatDoNotExist
         }
-
-        val sameIdsThatDoNotExist =
-          Seq(externalIdsThatDoNotExist.head, externalIdsThatDoNotExist.head)
-        val sameIdsThrown = the[CdpApiException] thrownBy deletable.deleteByExternalIds(
-          sameIdsThatDoNotExist
-        )
-        if (supportsMissingAndThrown) {
-          // as of 2019-06-03 we're inconsistent about our use of duplicated vs missing
-          // if duplicated ids that do not exist are specified.
-          val sameMissingIds = sameIdsThrown.duplicated match {
-            case Some(duplicatedIds) =>
-              duplicatedIds.flatMap(jsonObj => jsonObj("externalId").get.asString)
-            case None =>
-              sameIdsThrown.missing
-                .getOrElse(Seq.empty)
-                .flatMap(jsonObj => jsonObj("externalId").get.asString)
+        if (trySameIdsThatDoNotExist) {
```

Comment:
@hakontro  Finding this out was painful. I spent the whole day since I couldn't manage running tests in debug mode and could not print to stdout in tests. At the end, I sent requests for different data classes and found out that relationships give a different response in this case (explained). This part works for all the others the same. :( I would appreciate if you can show me if there is a proper way of debugging.